client_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -125,7 +125,8 @@ func (d *VolumeDriver) List(r volume.Request) volume.Response {
 // GetVolume - return volume meta-data.
 func (d *VolumeDriver) GetVolume(name string) (map[string]interface{}, error) {
 	mdata, err := d.ops.Get(name)
-	if err != nil {
+	// Get for empty volume name is issued by docker when it is coming up. Do not log error for this case.
```

Comment:
Why even make a trip to ESX for empty volume name?

Reply:
Yes. Pushing the next diff.